wavefront_sdk/common/utils.py

```diff
@@ -352,7 +352,7 @@ def event_to_json(name, start_time, end_time, source, tags,
     if not source:
         source = default_source
 
-    event = {'name': name, 'annotations': {}, 'hosts': source}
+    event = {'name': name, 'annotations': {}, 'host': source}
```

Comment:
For the events API JSON, the field is in fact called `hosts` and it takes an array of strings.

See the [Go SDK](https://github.com/wavefrontHQ/wavefront-sdk-go/blob/76e72d70ed85fdfe22217b6ddaa16464fa641fc5/senders/formatter.go#L279) for example.